classes/Validate.php

```diff
@@ -1331,7 +1331,7 @@ public static function isControllerName($name)
 
     public static function isPrestaShopVersion($version)
     {
-        return preg_match('/^[0-1]\.[0-9]{1,2}(\.[0-9]{1,2}){0,2}$/', $version) && ip2long($version);
+        return preg_match('/^[0-9]\.[0-9]{1,2}(\.[0-9]{1,2}){0,2}$/', $version) && ip2long($version);
```

Comment:
```suggestion
        return preg_match('/^[0-9]{1,2}\.[0-9]{1,2}(\.[0-9]{1,2}){0,2}$/', $version) && ip2long($version);
```

For the future :smile: